### PR TITLE
issho recipe

### PR DESCRIPTION
**Pull request by michaelbilow**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/issho/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 430394f679ca5bedf603c4f68ec576149a90de1f70bbb82af2e50fa8c634bf06 
+
+build:
+  skip: true  # [py<34]
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  build:
```
Comment (chrisburr) [inline recipes/issho/meta.yaml]: You don't need any `build` requirements so you can remove these three lines

File: recipes/issho/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 430394f679ca5bedf603c4f68ec576149a90de1f70bbb82af2e50fa8c634bf06 
+
+build:
+  skip: true  # [py<34]
```
Comment (chrisburr) [inline recipes/issho/meta.yaml]: This looks like it could be made noarch?
```suggestion
  noarch: python
```

File: recipes/issho/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  build:
+    - python
+    - pip
+  host:
+    - python
```
Comment (chrisburr) [inline recipes/issho/meta.yaml]: If it is made `noarch` you can constrain the Python version with:
```suggestion
    - python >=3.5
```

File: recipes/issho/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+    - pip
+  run:
+    - python
```
Comment (chrisburr) [inline recipes/issho/meta.yaml]: Similarly:
```suggestion
    - python >=3.5
```

File: recipes/issho/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+    - issho
+
+about:
+  home: http://github.com/michaelbilow/issho
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: issho allows simple access to commands on a remote machine over ssh from Python
+
+  # The remaining entries in this section are optional, but recommended
```
Comment (chrisburr) [inline recipes/issho/meta.yaml]:
```suggestion
```

File: recipes/issho/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+    issho and intuitive wrapper over paramiko for configuring and talking to a remote host. 
+    keyring is used to manage secrets locally.
+    issho is designed such that interacting with a single, heavily used remote machine 
+    should be easy, and working with more than one remote machine should be simple.
+  doc_url: http://issho.readthedocs.io/
+  dev_url: https://github.com/michaelbilow/issho
+
+extra:
+  recipe-maintainers:
+    # GitHub IDs for maintainers of the recipe.
```
Comment (chrisburr) [inline recipes/issho/meta.yaml]:
```suggestion
```

File: recipes/issho/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+    # Always check with the people listed below if they are OK becoming maintainers of the recipe. (There will be spam!)
```
Comment (chrisburr) [inline recipes/issho/meta.yaml]:
```suggestion
```

**Comment by michaelbilow** [comment]
@chrisburr thanks for the comments; I tried making the package noarch earlier at 0237f56, but got complaints from the CI about the windows build that I didn't understand. I'll make these changes and see how it goes.

**Comment by michaelbilow** [comment]
@chrisburr it's the same error on the Azure windows build as before that I don't understand.

```
Packaging issho
INFO:conda_build.build:Packaging issho-0.3.5-py_0
Packaging issho-0.3.5-py_0
C:\ProgramData\Chocolatey\lib\mingw\tools\install\mingw64\x86_64-w64-mingw32\bin\nm.exe: C:/_work/11/b/tcl-core-8.6.9.0/win/Release_AMD64_VC13/tcl_ThreadedDynamic/tclOOStubLib.obj: File format not recognized
C:\ProgramData\Chocolatey\lib\mingw\tools\install\mingw64\x86_64-w64-mingw32\bin\nm.exe: C:/_work/11/b/tcl-core-8.6.9.0/win/Release_AMD64_VC13/tcl_ThreadedDynamic/tclTomMathStubLib.obj: File format not recognized
C:\ProgramData\Chocolatey\lib\mingw\tools\install\mingw64\x86_64-w64-mingw32\bin\nm.exe: C:/_work/11/b/tcl-core-8.6.9.0/win/Release_AMD64_VC13/tcl_ThreadedDynamic/tclStubLib.obj: File format not recognized
C:\ProgramData\Chocolatey\lib\mingw\tools\install\mingw64\x86_64-w64-mingw32\bin\nm.exe: C:/_work/11/b/tk-8.6.9.0/win/Release_AMD64_VC13/tk_ThreadedDynamic/ttkStubLib.obj: File format not recognized
C:\ProgramData\Chocolatey\lib\mingw\tools\install\mingw64\x86_64-w64-mingw32\bin\nm.exe: C:/_work/11/b/tk-8.6.9.0/win/Release_AMD64_VC13/tk_ThreadedDynamic/tkStubLib.obj: File format not recognized
number of files: 14
WARNING (issho,Scripts\issho.exe): Needed DSO C:\Windows\System32\SHLWAPI.dll not found in any CDT/compiler package, nor the whitelist?!
   INFO (issho,Scripts\issho.exe): Needed DSO $SYSROOT\System32\KERNEL32.dll found in the whitelist
[noarch_python] Noarch package contains binary script: issho.exe
##[error]Cmd.exe exited with code '1'.
##[section]Finishing: Build recipe (maybe vs2008)
```

**Comment by chrisburr** [comment]
Yeah that error is caused by missing entry_points. LGTM 👍

**Comment by michaelbilow** [comment]
Awesome, thank you! One last question--I know that two of the packages (paramiko and cryptography) [conflict with each other](https://github.com/paramiko/paramiko/issues/1369) based on the apparent build recipe. 

I fixed it in the package's `requirements.txt` but I'm not sure how that should be dealt with in the `requirements.run` section of the recipe's `meta.yaml`.

**Pull request merged**